scripts/deploy-contracts-live.sh

```diff
@@ -107,9 +129,10 @@ deploy_contracts() {
   local swap_tree="${16}"
   swap_tree=$(echo "$swap_tree" | sed 's/^"\(.*\)"$/\1/')
 
-  local -r protocol="$dex-$protocol_currency"
+  local protocol="${network}-${dex}-${protocol_currency}-333"
+  protocol="${protocol^^}"
 
-  FLAGS="--broadcast-mode block --fees 200000unls --gas auto --gas-adjustment 1.2 --node $nolus_net --chain-id $chain_id"
+  FLAGS="--fees 200000unls --gas auto --gas-adjustment 1.2 --node $nolus_net --chain-id $chain_id"
```

Comment:
ok, 'block' is no more supported but it would be great to explicitly specify the new mode we want to use

Reply:
the default value is the one we use

Reply:
the default value may change hence we prefer to be explicit and specify the one we want